### ISSUE TITLE
Could not load npm packages which uses ASYNC AWAIT

### ISSUE THREAD

**Issue by mburger81** [driftyco/ionic-app-scripts#989]
#### Short description of the problem:
We load some npm packages/libs, which are written in javascript, this for example is the lib I'm talking about: https://github.com/Streamedian/html5_rtsp_player/
There are also another bug we resolved by our own and for not it is not included in the github repo.
Anyway the problem is, they are using ASYNC and AWAIT in some of there functions, and this is breaking the app.

As I know this is es6 standard and is not supported so for angular/ionic, so is there a possibility to support it, and if not could this be a feature request? Are there any plans to pass to es6?

**Comment by danbucholtz** [comment]
For development builds, we do not transpile code down to es5 for performance reasons. It takes too long to do.

For prod builds, we do. Is this getting converted for the prod build?

Typically libraries do not ship es201x code on npm. For example, `ionic` and `angular` both ship as `es5`.

Thanks,
Dan

**Comment by mburger81** [comment]
You are right. But there are some libs which are 3rd part and they don't care about that like angular and ionic does. 
What if is so? 
The lib is to complicate to convert it manually and using online convert is a solution where he had always problema before. 

We are using this lib in dev and production mode. For no we tried to get it work only on dev mode, we need it on dev before we can release it on production. 

We also tried to convert it to typescript but also this would be too much work? 

Perhapase you could add an feature to configure single file to be transpiled down to es5 also in dev mode?
